industrial_ci/rosdep-install.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -x
+#!/bin/bash -ex
```

Comment:
I opened https://github.com/ipa-mdl/industrial_ci/issues/3 to show what I will probably want to see in the future :)